### ISSUE TITLE
Geometry of glass in pouring task

### ISSUE THREAD

**Issue by ap229997** [Xingyu-Lin/softgym#28]
Is it possible to change the geometry of the glass (different shapes like cylinder or something like a cup/bowl) in the pouring task?

**Comment by DanielTakeshi** [comment]
Yes, you can do that. You can import different meshes (e.g., designed from Blender), adjust the length of the boxes, etc. What specifically would you like to do?

**Comment by ap229997** [comment]
The main goal is to learn pouring policies that can handle variation in the geometry of the cup/glass. For that, I need to run simulation with different objects. So, I want to import obj files from ShapeNet (or any other dataset) into SoftGym.

**Comment by NiteeshMidlagajni** [comment]
> Yes, you can do that. You can import different meshes (e.g., designed from Blender), adjust the length of the boxes, etc. What specifically would you like to do?

Api to add custom mesh is not available currently right? Do you perhaps have a working environment where meshes are present?

**Comment by scarlett-sun** [comment]
yeah i would also like to know how to import meshes and robot arms into softgym
